prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -102,7 +102,8 @@ spec:
           type: spc_t
       volumes:
         - name: weavedb
-          emptyDir: {}
+          hostPath:
+            path: /var/lib/weave-net
```

Comment:
I think the directory `/var/lib/weave` is fine, but we could use the `--db-prefix` setting to put "net" into the filename.

I guess there are few enough plugin-v2 users that we don't mind disrupting them slightly